core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -264,24 +291,87 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     const response = await this.threadDataBackendApiService.fetchMessagesAsync(
       threadId);
     const threadMessageBackendDicts = response.messages;
-    this.reviewMessage = threadMessageBackendDicts.map(
-      m => ThreadMessage.createFromBackendDict(m))[1].text;
+    let threadMessages = threadMessageBackendDicts.map(
+      m => ThreadMessage.createFromBackendDict(m));
+    if (threadMessages[1] !== undefined) {
+      this.reviewMessage = threadMessages[1].text;
+    }
```

Comment:
Hmm, can you explain this?

Reply:
This check is needed to prevent a console error when opening a contribution which has no review message. When a contribution has a review message the threadMessages array will contain 2 elements, the first element contains initial state of the contribution (thus an empty review message) and the second element contains the actual review message.

Reply:
Ah okay, can you add a comment about this?

Reply:
Done